api/users/serializers.py

```diff
@@ -102,6 +102,8 @@ def update(self, instance, validated_data):
                 setattr(instance, attr, value)
         try:
             instance.save()
+        except ValidationError as e:
+            raise InvalidModelValueError(detail=e.args)
```

Comment:
Oh, I didn't realize `e.args[0]` was a `dict`. Why is it a `dict` if you're passing a string [here](https://github.com/CenterForOpenScience/osf.io/pull/7478/files#diff-816963fa1deefc5403af7aadd46d0898R88)?